projects/storefrontapp/src/app/spartacus/spartacus-features.module.ts

```diff
@@ -105,7 +106,11 @@ if (environment.cpq) {
   featureModules.push(ProductConfiguratorRulebasedFeatureModule);
 }
 if (environment.digitalPayments) {
-  featureModules.push(DigitalPaymentsFeatureModule);
+  if (environment.b2b) {
+    featureModules.push(CheckoutReplenishmentDpFeatureModule);
+  } else {
+    featureModules.push(DigitalPaymentsFeatureModule);
+  }
```

Comment:
Didn't we agree to have a "wrapper" module only for _one_ checkout? Either the base, b2b _or_ repl?

Do we need to have three module in our sample app?